specification/ai/Azure.AI.Projects/agents/tools/models.tsp

```diff
@@ -128,6 +138,20 @@ model FunctionDefinition {
   parameters: unknown;
 }
 
+@doc("The definition of Azure function.")
+model AzureFunctionDefinition {
+
+  @doc("The definition of azure function and its parameters.")
+  function: FunctionDefinition;
+
+  @doc("Input storage queue.")
```

Comment:
Can we add more information about storage queue's for users?

Reply:
Fixed.